run.py

```diff
@@ -17,37 +17,33 @@
 pygame.mixer.init()
 pygame.mixer.music.load(MUSIC_FILE_PATH)
 pygame.mixer.music.set_volume(MUSIC_VOLUME_PERCENTAGE)
-# the -1 argument repeats the song endlessly
-pygame.mixer.music.play(-1)
+pygame.mixer.music.play(-1) # -1 to repeat song endlessly
 
 
 #tank instance
 player = Player(400, 300, 32, 32)
-#append missile
 player_missile = []
 
 #Add a target sprite with random size
 shootingTarget = Target()
 targetSprites.add(shootingTarget)
 
-#timer instance
-# timer_countdown = Timer()
+#set up background object and skip menu background
+game_background = Background(BACKGROUND_IMAGES_FILE_PATHS)
+game_background.increment_level_background()
```

Comment:
Looks good! So is the Menu image blitting over the game play background until game_state == START_GAME?

Reply:
No, I don't think so.  The menu.image should be blitting onto the display surface until START_GAME, then the game play background will get blitted onto the display surface.